contrib/metrics-exporter/README.md

```diff
@@ -0,0 +1,8 @@
+# CRI-O - Metrics Export
+
+Dockerfile for metrics export can be found [here](Dockerfile)
```

Comment:
nit: I think this should probably be Containerfile instead

Reply:
In that case, I feel we should rename this Dockerfile as `Containerfile` and delete the existing [Containerfile](https://github.com/amarlearning/cri-o/blob/main/contrib/metrics-exporter/Containerfile). What do you think?

Reply:
I'm guessing that was a desire of @djkormo . I am open either way, it would probably be cleaner to just have the Containerfile but if there is desire for a Dockerfile then :shrug:

Reply:
@cri-o/cri-o-maintainers WDYT?

Reply:
I don't see a need for Dockerfile as of now. And keep both file is violating DRY.

Reply:
wfm!

Reply:
I second @amarlearning opinion.

Reply:
Thanks @sohankunkerkar @haircommander

I have removed the Dockerfile